pages/common/rename.md

```diff
@@ -10,7 +10,7 @@
 
 `rename -n {{'s/foo/bar/'}} {{\*}}`
 
-- Convert filenames to lower case:
+- Convert filenames to lower case (use `-f` in case-insensitive filesystems to prevent "already exists" errors):
```

Comment:
Yes, but then it wouldn't really apply to the case changing the case of a file in case-insensitive filesystems, since the target filename doesn't actually exist. We could add an example with -f before this one to ensure it gets a proper explanation -- would that work?